advocacy_docs/supported-open-source/postgresql/uninstalling.mdx

```diff
@@ -51,15 +52,13 @@ These instructions apply if you used EDB's interactive Mac installer to install
 
 To uninstall PostgreSQL on a Mac system:
 
-1.  Assume the identity of an operating system superuser, and navigate into the folder where the uninstaller resides:
-
-  `/Library/PostgreSQL/<version>`
-
-  Where `version` is the version of PostgreSQL you installed.
-
 1.  Invoke the uninstaller:
 
-  `open uninstall-postgresql.app`
+    ```
+    sudo /Library/PostgreSQL/<version>/uninstall-postgresql.app/Contents/MacOS/installbuilder.sh
```

Comment:
```suggestion
    open /Library/PostgreSQL/<version>/uninstall-postgresql.app
```

Reply:
This is not the appropriate way to invoke the uninstaller. It blocks the ability for users to use a safe admin account on the mac.